tachyon_core/src/storage/hash_map.rs

```diff
@@ -81,8 +81,8 @@ impl<V: Copy> IDLookup<V> {
             unsafe {
                 drop(Box::from_raw(cur));
             }
+            self.size -= 1;
```

Comment:
Unrelated change - the precondition to using IDLookup is that you only remove keys that exist, but there was no harm in making this safe